src/olympia/devhub/tasks.py

```diff
@@ -821,6 +821,32 @@ def resize_preview(src, preview_pk, **kw):
         log.error("Error saving preview: %s" % e)
 
 
+@task
+def recreate_previews(addon_pk, **kw):
+    log.info('[1@None] Recreating previews for addon: %s' % addon_pk)
+    addon = Addon.objects.get(pk=addon_pk)
+    for preview in addon.previews.all():
+        log.info('[1@None] Resizing preview: %s' % preview.id)
+        try:
+            if not preview.sizes:
+                preview.sizes = {}
+            if storage.exists(preview.original_path):
+                # We have an original size image, so we can resize that.
+                src = preview.original_path
+                (preview.sizes['image'], _) = resize_image(
+                    src, preview.image_path, amo.ADDON_PREVIEW_SIZES['full'])
```

Comment:
Note that we discussed this during the all hands: Make sure that we issue a blog post for developers to update/re-upload their screenshots if possible.

Reply:
#7908 is part of #7907 and that issue mentions Product will communicate changes.